src/Autonomous.cpp

```diff
@@ -51,6 +55,20 @@ const transform_t VIZ_BASE_TF =
  */
 static bool printLandmarks(const points_t& landmarks, int log_level = LOG_DEBUG);
```

Comment:
Is this form of `printLandmarks` still used anywhere?

Reply:
Some planner code solves for landmark locations as a vector of points, so this definition of the method is used to print those landmarks that were solved for. It's not great since ideally we want to deprecate the use of zero landmarks, but I didn't want to either convert from a vector of points to a landmarks_t, or I didn't want to refactor the planner code, so I just left it as-is.